neopia/neosoco.py

```diff
@@ -701,6 +701,12 @@ def servo_rotate_by_degree(self, port='out1', direction='forward', speed='100',
         else:
             raise TypeError
 
+    def servo_stop(self, port = 'out1'):
+        if isinstance(port, str):
+            self._write_to_output_port(port, 0)

```

Comment:
For the stop, the value should be 254 like mentioned here: https://github.com/entrylabs/entryjs/blob/b7eb3e0a5fcb37fc7451f0c8ac1fa5657c1f8992/src/playground/blocks/hardware/block_neobot_purple.js#L2164